.travis.yml

```diff
@@ -10,4 +10,10 @@ notifications:
     on_success: change  # options: [always|never|change] default: always
     on_failure: always  # options: [always|never|change] default: always
     on_start: false     # default: false
+env:
+  global:
+  - secure: STYk3FPnQwBHT25T.../O2SWbVYIL74K...XQAw=
+  - secure: MwRqcAgpBGrmyRY6.../lB/sjDhMk67Y...Xjgk=
 
+after_success:
+  - "[[ $TRAVIS_BRANCH == \"master\" ]] && { sbt publish; };"
```

Comment:
Shamelessly [taking from Shapeless], we might want a `"${TRAVIS_PULL_REQUEST}" == "false"` here at least.

  [taking from Shapeless]: https://github.com/milessabin/shapeless/blob/a1baba9acb06d6be63b501656ee3a851da9b3244/.travis.yml#L15